cache/api/src/main/java/org/apache/karaf/cache/api/CacheService.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.karaf.cache.api;
+
+import javax.cache.configuration.Configuration;
+import java.util.List;
+
+public interface CacheService {
+
+    <K, V> void createCache(String name, Configuration<K, V> configuration);
+
+    <K, V> V get(String name, K key);
+
+    <K, V> void put(String name, K key, V value);
+
+    void invalidateCache(String name);
+
+    List<String> listCaches();
+}
```

Comment:
A method to get the entire JSR107 `Cache` (other than get/put) might be useful too.